src/adapters/address-adapter.ts

```diff
@@ -0,0 +1,62 @@
+// Declare Address type for object returned from backend API
+type Address = {
+    address: string,
+    latitude: number,
+    longitude: number,
+}
+
+/**
+ * This function will return the best match for given address with coordinates.
+ * @param {String} address - address to search.
+ * @example
+ * const address = "315 Kildaire Farm Rd, Cary, NC 27511";
+ * getAddressCoordinate(address);
+ * @returns {Promise<void | Address>} A promise that is top matching result for given address with coordinates..
+ */
+export const getAddressCoordinate = (address: string): Promise<void | Address> => 
```

Comment:
Can you convert both adapters to async/await for consistency with the other adapters?

Reply:
done